bears/csv/CSVLintBear.py

```diff
@@ -1,11 +1,33 @@
 from coalib.bearlib.abstractions.Linter import linter
 from coalib.bears.requirements.GemRequirement import GemRequirement
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
 
 
 @linter(executable='csvlint',
         output_format='regex',
-        output_regex=r'\d\. (?P<message>.+(s|g|e|w|d)\.*)'
-                     r'( |$)(?P<additional_info>.*)')
+        output_regex=r'\n[0-9]+\.\s(?P<origin>(?P<severity>\w+))\.\s'
```

Comment:
I notice you've added a \n ... is that required ? Seems weird because the first line will be ignored then right ?